src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -34,6 +34,7 @@
 
     private Inventory inventory;
     private int size;
+    private final static String backpacks = "backpacks.";
```

Comment:
Wrong modifier order, this should be `private static final`.

Reply:
ff83a0183